backend/uclapi/resources/views.py

```diff
@@ -39,7 +39,7 @@ def get_pc_availability(request, *args, **kwargs):
         _ = pc.get
         data.append({
             "location": {
-                "room_name": _("location"),
+                "roomname": _("location"),
```

Comment:
Any reason for this change?

Reply:
yep! cos we use roomname in other parts of the api